content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,79 @@
+## Installation
+
+- **Added** experimental [manifest and profile commands](/docs/setup/install/operator/) to install and manage the Istio control plane for evaluation.
+
+## Traffic management
+
+- **Added** [automatic determination](/docs/ops/traffic-management/protocol-selection/) of protocol types. You no longer need to explicitly prefix service ports with their protocol type.
+- **Added** a mode to the Gateway API for mutual TLS operation.
+- **Fixed** issues present when a service communicates over the network first in permissive mutual TLS mode for protocols like MySQL and MongoDB.
+- **Improved** Envoy proxy readiness checks. They now check Envoy's readiness status.
+- **Improved** container ports are no longer required in the pod spec. All ports are [captured by default](/faq/traffic-management/#controlling-inbound-ports).
+- **Improved** the `EnvoyFilter` API. You can now add or update all configurations.
+- **Improved** the Redis load balancer to now default to [`MAGLEV`](https://www.envoyproxy.io/docs/envoy/v1.6.0/intro/arch_overview/load_balancing#maglev) when using the Redis proxy.
+- **Improved** load balancing to direct traffic to the [same region and zone](/faq/traffic-management/#controlling-inbound-ports) by default.
+- **Improved** Pilot by reducing CPU utilization. The reduction approaches 90% depending on the specific deployment.
+- **Improved** the `ServiceEntry` API to allow for the same hostname in different namespaces.
+- **Improved** the [Sidecar API](/docs/reference/config/networking/v1alpha3/sidecar/#OutboundTrafficPolicy) to customize the `OutboundTrafficPolicy` policy.
+
+## Security
```

Comment:
@myidpt can we order these items giving priority to primary user facing features and then debuggability and other maintainability related improvements

Reply:
@louiscryan this is an unordered list.  If you wish to reorder for highlighting that wfm.  Pleawse note, the format of the bolded is ordered as per decision in docs wg.

Cheers
-steve

Reply:
@sdake Please update the ordering as below, thanks a lot.

- **Added** trust domain validation for services using mutual TLS. By default, the server only authenticates the requests from the same trust domain.
- **Added** [labels](/docs/concepts/security/#how-citadel-determines-whether-to-create-service-account-secrets) to control service account secret generation by namespace.
- **Added** SDS support to deliver the private key and certificates to each Istio control plane service.
- **Added** support for [introspection](/docs/ops/troubleshooting/controlz/) to Citadel.
- **Added** metrics to the `/metrics` endpoint of Citadel Agent on port 15014 to monitor the SDS service.
- **Added** diagnostics to the Citadel Agent using the `/debug/sds/workload` and `/debug/sds/gateway` on port 8080.
- **Improved** the ingress gateway to [load the trusted CA certificate from a separate secret](/docs/tasks/traffic-management/ingress/secure-ingress-sds/#configure-a-mutual-tls-ingress-gateway) when using SDS.
- **Improved** SDS security by enforcing the usage of [Kubernetes Trustworthy JWTs](/blog/2019/trustworthy-jwt-sds).
- **Improved** Citadel Agent logs by unifying the logging pattern.
- **Removed** support for Istio SDS when using [Kubernetes versions earlier than 1.13](/blog/2019/trustworthy-jwt-sds).
 - **Removed** integration with Vault CA temporarily. SDS requirements caused the temporary removal but we will reintroduce Vault CA integration a future release.
- **Enabled** the Envoy JWT filter by default to improve security and reliability.

Reply:
accepted.

Reply:
Could you add one more thing:
```
- **Removed** the Mixer RBAC adapter which has been deprecated since 1.1.
```